app/models/pawn.rb

```diff
@@ -1,2 +1,38 @@
 class Pawn < Piece
+  def legal_move?(x, y)
```

Comment:
All looks good.

This method name should be `valid_move?` to match the other pieces (since we'll be calling the same method for each piece type)